packages/baggage-span-processor/README.md

```diff
@@ -46,6 +46,16 @@ const sdk = new NodeSDK({
 sdk.start();
 ```
 
+A predicate function that determines what baggage entries are copied can be provided as a constructor parameter.
```

Comment:
In other READMEs in this repo, we usually have a "Component Options" section which explains each options. something like:

```
### Baggage SpanProcessor Options
```

And then describe the option/s one by one, possibly in a markdown table (but maybe overkill for one option).